packages/app-desktop/gui/EncryptionConfigScreen/EncryptionConfigScreen.tsx

```
@@ -354,7 +354,10 @@ const EncryptionConfigScreen = (props: Props) => {
 				<h2>{_('Re-encryption')}</h2>
 				<p style={theme.textStyle} dangerouslySetInnerHTML={{ __html: t }}></p>
 				<span style={{ marginRight: 10 }}>
-					<button onClick={() => void reencryptData()} style={theme.buttonStyle}>{buttonLabel}</button>
+					<button onClick={async () => {
+						await reencryptData(msg => dialogs.confirm(msg), msg => dialogs.alert(msg));
+						window.focus();
```

ESLint

[error] 'focus' is restricted from being used. Please use focusHandler::focus() instead

(no-restricted-properties)